examples/component/map.js

```diff
@@ -1,58 +1,49 @@
 Ext.require([
-    'GeoExt.component.Map',
-    'Ext.panel.Panel',
-    'Ext.Viewport'
+    "GeoExt.component.Map",
+    "Ext.panel.Panel",
+    "Ext.Viewport"
 ]);
 
-var olMap,
-    mapComponent,
-    mapPanel;
```

Comment:
Especially in the examples the declaration of the vars outside of the application has been done by purpose to allow users easily check they main instances in the console. So here it might by OK having it this way?